test/minting/MintP0_ArgumentTests.js

```diff
@@ -166,18 +166,8 @@ async function run_MINT_tests(newToken, MintControllerArtifact, accounts) {
         await checkMINTp0([token, mintController], [expectedTokenState, expectedMintControllerState]);
     });
 
-    it('arg015 incrementMinterAllowance(0) makes no changes to allowance', async function () {
-        var amount = 897;
-        await mintController.configureController(Accounts.controller1Account, Accounts.minterAccount, {from: Accounts.mintOwnerAccount});
-        await mintController.configureMinter(amount, {from: Accounts.controller1Account});
-        await mintController.incrementMinterAllowance(0, {from: Accounts.controller1Account});
-
-        expectedMintControllerState.controllers['controller1Account'] = Accounts.minterAccount;
-        expectedTokenState.push(
-            { 'variable': 'isAccountMinter.minterAccount', 'expectedValue': true },
-            { 'variable': 'minterAllowance.minterAccount', 'expectedValue': newBigNumber(amount) }
-        );
-        await checkMINTp0([token, mintController], [expectedTokenState, expectedMintControllerState]);
+    it('arg015 incrementMinterAllowance(0) throws', async function () {
+        await expectRevert(mintController.incrementMinterAllowance(0, {from: Accounts.controller1Account}));
```

Comment:
Use `expectError` so we can check for the correct error message.  You will probably need to configure the controller first - otherwise you will get an error that only controllers can increment the minter allowance.